producers/smd_producer.py

```diff
@@ -237,8 +239,8 @@ def global_except_hook(exctype, value, exc_traceback):
 from smalldata_tools.ana_funcs.smd_svd import svdFit
 from smalldata_tools.ana_funcs.correlations.smd_autocorr import Autocorrelation
 
-# logging.basicConfig(level=logging.DEBUG)
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Going to DEBUG used to print _a lot_ of detector information per rank. What makes the change of the default necessary?

Reply:
Yes I have that as I am still working on it. I will go back to info later. Keeping this comment open as a reminder to mysefl :-)